packages/login/README.md

```diff
@@ -0,0 +1,54 @@
+# Login
+
+App to authenticate with nice2 services, request or update password.
+Password-update can be embedded independently.
+
+##Login-Form
+
+Name: login
+
+### Input parameters
+
+| Name                   | Mandatory | Description
+|------------------------|-----------|-------------
+| `showTitle`            |           | Display a title above the dialog.
+| `locale`               |           | ISO Language Code
+
+
+### Methods
+
+| Name                   | Params    | Description
```

Comment:
Even when this table looks fine when it is rendered, I would prefer if indention of last cell would be fixed to look also fine in text format. This applies also to other tables in this change.